tests/e2e_tests/subcommands/root/test_root_boost_weights.py

```diff
@@ -0,0 +1,59 @@
+from bittensor.commands.root import RootSetBoostCommand, RootSetWeightsCommand
+from bittensor.commands.stake import StakeCommand
+from bittensor.commands.unstake import UnStakeCommand
+from bittensor.commands.network import RegisterSubnetworkCommand
+from bittensor.commands.register import RegisterCommand
+from ...utils import new_wallet, sudo_call_set_network_limit
+import bittensor
+
+
+# we can't test it now since the root network weights can't be set
```

Comment:
If we cant test this right now, should we remove it?

Reply:
pls keep it in the repo. I need some time to try, then know what's the correct parameter and how to check.

Reply:
Can you keep this in a separate branch? There is no need to push this in while its still under development and not ready to go out to the public. Every test we run on the Bittensor side actually tests what it is supposed to test. This will add tests that don't actually test what they claim to test, and we will be required to "remember" that X or Y test is incomplete, giving us holes in what bugs and issues can go unnoticed.